### PR TITLE
[tests] Update 2 periodically failing tests

### PR DESCRIPTION
**Pull request by cidrblock**
The templar integration tests fail periodically, this _should_ fix that.

Since the file is `:opened` vim might be leaving a swap file:

```
'Found a swap file by the name '
 '"~/src/github.com/ansible/ansible-navigator/tests/fixtures/integration/actions/run/.site.yaml.swp"',
 '          owned by: zuul   dated: Tue Dec  7 18:52:13 2021',
 '         file name: '
 '~zuul/src/github.com/ansible/ansible-navigator/tests/fixtures/integration/actions/run/site.yaml',
 '          modified: no',
 '         user name: zuul   host name: '
 'centos-8-stream-vexxhost-sjc1-000204357',
 '        process ID: 76326 (still running)',
 'While opening file '
```

The inventory integration tests fail periodically, this _may_ fix that.

- additionally add some look_fors to failing inventory test (as a test)

### CODE REVIEW

**Comment by cidrblock** [comment]
recheck

**Review by webknjaz — APPROVED** [review]

**Comment by cidrblock** [comment]
recheck

**Comment by cidrblock** [comment]
Recheck

**Comment by cidrblock** [comment]
Recheck

**Comment by cidrblock** [comment]
Recheck

**Comment by cidrblock** [comment]
Recheck

**Comment by cidrblock** [comment]
recheck

File: tests/integration/actions/templar/base.py [hunk 1]
```diff
@@ -50,13 +50,17 @@ class BaseClass:
     @staticmethod
     @pytest.fixture(scope="module", name="tmux_session")
     def fixture_tmux_session(request):
-        """tmux fixture for this module"""
+        """tmux fixture for this module
```
Comment (webknjaz) [inline tests/integration/actions/templar/base.py:53]:
```suggestion
        """Return a new tmux session.
```
(this is also what's displayed in the output of `pytest --fixtures`)

Comment (cidrblock) [reply tests/integration/actions/templar/base.py:53]: Sorry, out of scope here. Linting and doc string will slowly be fixed in other PRs.

Comment (webknjaz) [reply tests/integration/actions/templar/base.py:53]: (ignoring as out of the scope)

**Comment by cidrblock** [comment]
recheck

**Review by ssbarnea — APPROVED** [review]

**Comment by cidrblock** [comment]
Force merged due to other failures.

**Pull request merged**